addon/lib/Feature.jsm

```diff
@@ -695,9 +726,11 @@ class Feature {
       (panelHiddenTime - this.panelShownTime) / 1000;
     this.log.debug(`${panelType} was open for ${Math.round(panelOpenTime)} seconds.`);
     this.telemetry({
-      event: `${panelType}-hidden`,
-      secondsPanelWasShowing: Math.round(panelOpenTime).toString(),
-    });
+        message_type: "event",
+        event: "panel-hidden",
+        panel_type: panelType,
+        secondsPanelWasShowing: Math.round(panelOpenTime).toString(),
```

Comment:
This is actually currently in ms, not s. I can change that easily.

Reply:
I'll change it.

Reply:
Oh are you sure about this? (panelHiddenTime - this.panelShownTime) / 1000 is seconds?

Reply:
Oh maybe not... hmm... what did I see that was in ms? Ah well. Please disregard then.